packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -99,17 +104,18 @@ class BpkDataTable extends Component {
   }
 
   onHeaderClick({ dataKey: sortBy, event }) {
-    const disableSort =
-      this.props.children.find(child => child.props.dataKey === sortBy).props
-        .disableSort === true;
-    if (disableSort) {
+    const children = this.props.children.find(
```

Comment:
I think it will be more clear if this variable is named `column` (instead of `children`)

Reply:
yes, makes sense